### PR TITLE
issue #31 - Conversion between Celsius and Fahrenheit

### PR DESCRIPTION
**Pull request by g3org3**
Created a simple way to change temperature from celsius to fahrenheit
screen shot 2016-10-09 at 6 44 22 pm

### CODE REVIEW

File: app/components/Weather.jsx [hunk 1]
```diff
@@ -19,14 +21,14 @@ var Weather = React.createClass({
       errorMessage: undefined,
       location: undefined,
       temp: undefined,
-			condition: undefined
+      condition: undefined
```
Comment (bmorelli25) [inline app/components/Weather.jsx:24]: Correct, no need for fix here

File: app/components/Weather.jsx [hunk 2]
```diff
@@ -19,14 +21,14 @@ var Weather = React.createClass({
...
     });
 
     openWeatherMap.getWeather('weather', location).then(currentWeather => {
       this.setState({
         location: currentWeather.name || location,
         temp: currentWeather.main.temp,
-				condition: currentWeather.weather[0],
+        condition: currentWeather.weather[0],
```
Comment (bmorelli25) [inline app/components/Weather.jsx:31]: Correct, no need for fix here

File: app/components/WeatherForm.jsx [hunk 3]
```diff
@@ -2,31 +2,60 @@ var React = require('react');
 var GoogleAutocomplete = require('./GoogleAutocomplete.jsx');
 
 var WeatherForm = React.createClass({
-  onFormSubmit: function(e) {
-    e.preventDefault();
-    var location = this.refs.googleAutoComplete.getLocation(); //set variable to location value
-    if (location.length > 0){ //check for input
+	onFormSubmit: function(e) {
```
Comment (bmorelli25) [inline app/components/WeatherForm.jsx:5]: tab/spacing issue

File: app/components/WeatherForm.jsx [hunk 4]
```diff
@@ -2,31 +2,60 @@ var React = require('react');
...
+		var celsiusChecked = { checked: this.props.tempType === 'C' }
+		var fahrenheitChecked = { checked: this.props.tempType === 'F' }
+		return (
+			<div>
+				<form onSubmit={this.onFormSubmit}>
+					<div>
 						<GoogleAutocomplete
 							style={{width: '90%'}}
-							ref="googleAutoComplete"
+							ref='googleAutoComplete'
```
Comment (bmorelli25) [inline app/components/WeatherForm.jsx:26]: undo this `"` change

File: app/components/WeatherMessage.jsx [hunk 5]
```diff
@@ -18,10 +18,16 @@ var WeatherMessage = React.createClass({
...
+
+    if (tempType === 'C') {
+      temp = (temp - 32) * (5/9)
+      temp = temp.toFixed(2)
+    }
+
     return (
-	  	<div>
-	      <h3 className="text-center">It is {temp}&deg; in {location}</h3>
+      <div>
```
Comment (bmorelli25) [inline app/components/WeatherMessage.jsx:29]: Correct, no need for fix here

File: app/components/WeatherMessage.jsx [hunk 6]
```diff
@@ -18,10 +18,16 @@ var WeatherMessage = React.createClass({
     };
   },
   render: function() {
-  	var {temp, location} = this.props;
+    var {temp, location, tempType} = this.props;
```
Comment (bmorelli25) [inline app/components/WeatherMessage.jsx:21]: Correct, no need for fix here

**Review by bmorelli25 — CHANGES_REQUESTED** [review]
Hey @g3org3! Thanks for the PR. I'm able to download and test on my end - and it looks like everything is working.

A few changes though. Please see my code comments. It looks like you're not using the `.editorconfig` file. Check the `readme` for more info on this. All tabs should be 2 spaces wide to maintain consistency across the files. Make these changes and I'll merge your PR!

Thanks again

**Comment by bmorelli25** [comment]
@g3org3 - it actually looks like you are using the editorconfig, my bad. Just a rouge spacing issue that I commented above. Also, please merge the master branch so the conflicts in `weatherMessage.jsx` are resolved as well!

File: app/components/WeatherForm.jsx (hunk 3 above)
Comment (g3org3) [reply app/components/WeatherForm.jsx:5]: there were spaces before so I changed them to tabs
screen shot 2016-10-10 at 11 26 28 am

**Comment by g3org3** [comment]
@bmorelli25 - I'll open another PR with master

**Comment by g3org3** [comment]
@bmorelli25 - There are still files with spaces rather tan tabs :( 
should we change all of the jsx files to tabs or leave that for another PR ?

**Pull request closed without merging**